docs/_data/navigation.yml

```diff
@@ -25,3 +25,5 @@ docs:
         url: /themes/
       - title: "Translation"
         url: /translation/
+      - title: "Release"
```

Comment:
I believe the title to be misdirecting.
When reading `Release` I think about a list of past releases, and the corresponding changelog.

Reply:
How should I name it?